src/Sarif/Writers/SarifLogger.cs

```diff
@@ -110,13 +116,18 @@ public class SarifLogger : IDisposable, IAnalysisLogger
                 (_dataToInsert & OptionallyEmittedData.BinaryFiles) != 0;
         }
 
-        private SarifLogger(TextWriter textWriter, LoggingOptions loggingOptions, bool closeWriterOnDipose)
+        private SarifLogger(
+            TextWriter textWriter,
+            LogFilePersistenceOptions logFilePersistenceOptions,
+            bool closeWriterOnDipose,
+            IEnumerable<FailureLevel> level,
```

Comment:
>level [](start = 38, length = 5)

nit: levels and kinds

Reply:
Done.

---
In reply to: [562032468](https://github.com/microsoft/sarif-sdk/pull/2241#discussion_r562032468) [](ancestors = 562032468)